docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
 
-* Some notes: XCTestCase is the XCTest subclass that contains our tests methods and XCTAssert lines are the expected results, with different types depending on what are you testing. Learn more about XCTAssert Methods and XCTesting itself <a href="http://rshankar.com/assertions-supported-in-xctest/" target="_blank">here</a> and <a href="http://nshipster.com/xctestcase/" target="_blank">here</a>.
+Create a new group folder called 'Support' inside the main group and move the unused files into it. Compare the image below to the one above. As with many tasks in Xcode, there is a shortcut. In this case Command-Click to select all the unneeded files, and choose `File -> New -> Group from Selection` from the main menu or use the contextual menu.
 
-* Now run the tests, and happy coding!
+![007-newprojectmidway](https://cloud.githubusercontent.com/assets/5385825/20957612/c769df04-bc05-11e6-9719-18b5aece0f2d.png)
 
-* When you are ready to submit, do so from the folder that contains the working file (`Bob.swift`) and the Exercism test file (`BobTest.swift`): 
-`$> exercism submit Bob.swift`
+Create a new playground document like in the image above. Although multiple menu choices are available, in order for this step to work properly use the main menu, `File -> New -> File...` (or Command-N) and choose Playground from the template chooser. (Other menu options tend to create the playground outside the project file.)
 
-\* Please note that I have used `bob-swift` as my example, to differentiate between the same exercise in multiple languages. If you hyphenate the project name, you will need to use an underscore instead of a hyphen in the name, in the test file. (`@testable import bob-swift` -> `@testable import bob_swift`)
+Once again in the image above, note the file `GigasecondTests.swift` inside the `GigasecondTests` folder. This file is unnecessary and can be deleted or moved somewhere else. Removing it is helpful because we are about to replace it with a new file of almost the same name.
 
-## Running Tests
+#### Import The Exercism Tests Source File
+![008-testsimportexample](https://cloud.githubusercontent.com/assets/5385825/20957618/c7800d4c-bc05-11e6-937f-de2f3743c1b6.png)
 
-Test will run through the Xcode GUI using the XCTest framework.
+Right-Click (or Control-Click) on the `GigasecondTests` folder and choose Add Files to "Gigasecond" from the contextual menu to open a file import dialog box. Navigate to and select the unit tests file in the exercise folder. In this example the file is named `GigasecondTest.swift`. Different exercises may vary the name format somewhat.
+
+Add the line `@testable import Gigasecond`. (If the Xcode project name contains a hyphen, replace it with an underscore. For example, with Word-count, enter `@testable import Word_count`). The red icons indicate failed tests or missing code objects, which is normal at this stage. Once imported correctly it should resemble the image above.
```

Comment:
I think that this last line is a bit confusing, as the red icons don't disappear once it is imported correctly. It continues showing red icons until you write a correct code in your "Application source file". We can say something like: 
The red icons indicate failed tests or missing code objects, which is normal at this stage. Once imported correctly, and once you write the solution of the exercise in the 'application source file', they should disappear.